### PR TITLE
Rework of #8116 - Update Amazon Macie Rules

### PR DESCRIPTION
**Pull request by fabamatic**
## Description
|Related issue|Related PR|Closes Issue|
|---|---|---|
| #8116 | #8783    |#11275|

## Changes & Comments
The aim of this PR is:

- Add Copyright header
- To ensure the rules and decoders from issue #8116 work correctly.
- To fix issues regarding indentation, syntax, rule granularity, spellings and ensure the ruleset quality is standard.

### Syntax

- [x] 1.a Rule tags order must be compliant.
- [x] 1.b Decoder tags order must be compliant.
- [x] 1.c XML blocks must as compact as possible.
- [x] 1.d Only one empty line between rule/decoder and the next rule/decoder.
- [x] 1.e Decoder extracted fields names must use '_' whether a space is needed.
- [x] 1.f Decoder name must use '-' whether a space is needed.
- [x] 2.d XML and ini files must use correct indentation
 
### Grammar

- [x] 2.a Grammar quality.
- [x] 2.b Similar phrases must keep tenses and expressions.
- [x] 2.c Grammar basic rules like capitalization, punctuation marks, sentence construction, and others.

### Semantic

- [x] 3.a New decoder, rule, or test are written in the correct file and grouped correctly inside the file.
- [x] 3.b Group similar rules under same ID's group.
- [x] 3.c: 
   - [x] 3.c1 Find and reuse group name before creating a new one.
   - [x] 3.c2 Include a new group definition in a PR comment.
   - [x] 3.c3 Any group name must use '_' whether it does need a space char.
   - [x] 3.c4 The groups inside a tag must be sorted in alphabetical order.
   - [x] 3.d Rule level should be compliant with the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html).

### Unit testing

- [x] 4.a A metadata block at the beginning of the .ini file describing software, version, and logs source.
- [x] 4.b Each new rule must have at least one test entry in the correct .ini file.
- [x] 4.c Runtest.py must pass and must include the results in raw format here.
<details>
  <statement>Unit Tests</statement>

  ```

  ```
</details>

- [x] 4.d New CDB lists must include the proper test entry in the correct .ini file.

### E2E testing

- ~~[] 5.a Logs for new or modified decoder/rules sent to a manager running with this PR ruleset appear in Kibana. ~~
- ~~[] 5.b There are not affected or broken visualizations on Kibana. ~~
-  ~~[] 5.c New or modified items can be seen correctly using APP ruleset navigation.~~

### Elasticsearch Template

- [x] 6.a Known fields with output format managed and usually used for searching are included in template array index.query.default_field. 
- [x] 6.b The new field with the correct date format is stored as a "date" type field in the template.
- [x] 6.c The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template. 
- [x] 6.d Known fields with output format managed and usually used for searching are included in the template.

### Stoppers

- [x] 7.a No previous rule ID changes without triple check.
- [x] 7.b No previous decoder name changes without triple check.
- [x] 7.c No previous file name changes without triple check.
- [x] 7.d No previous test changes its 'rule' field value without triple check.

### Others

- [x] 8.a Each file has the correct copyright block.
- [x] 8.b The copyright block doesn't have "Author" only "Created by Wazuh". To include an "Author" request triple check.
- [x] 8.c The copyright block doesn't use "-". 
- [x] 8.d The rule files don't have any sample log.
- [x] 8.e The decoder file has sample logs next to the decoder that matches that log.
- [x] 8.f The decoder and rule files have information about software, version, and any helpful information.
- [x] 8.g The PR has a single commit with CHANGELOG changes in the correct format.
- [x] 8.h The new rule ID is not in use.
- [x] 8.i The new rule ID must be in the defined IDs range. 
- [x] 8.j New rules ID range must be verified with a triple check and noted in the rules ID document.
- [x] 8.k The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template.

### CODE REVIEW

File: ruleset/rules/0350-amazon_rules.xml [hunk 1]
```diff
@@ -1,597 +1,632 @@
...
+
+  <!-- Events with errors -->
+  <rule id="80250" level="5">
+    <if_sid>80203</if_sid>
+    <field name="aws.errorCode">AccessDenied</field>
+    <options>no_full_log</options>
+    <description>AWS Cloudtrail: $(aws.eventSource) - $(aws.eventName). Error: $(aws.errorCode).</description>
+    <group>
+      aws_cloudtrail,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.6,nist_800_53_AU.14,nist_800_53_AC.7,pci_dss_10.6.1,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,tsc_CC7.2,tsc_CC7.3,tsc_CC6.1,tsc_CC6.8,
+    </group>
```
Comment (jcruzlp) [inline ruleset/rules/0350-amazon_rules.xml:61]: Is fine being split?

File: ruleset/rules/0350-amazon_rules.xml [hunk 2]
```diff
@@ -1,597 +1,632 @@
...
+    <if_sid>80350</if_sid>
+    <field name="aws.severity.description">CRITICAL</field>
+    <options>no_full_log</options>
+    <description>AWS Macie $(aws.severity.description): $(aws.name) - $(aws.summary.description)</description>
+    <group>aws_macie,</group>
+  </rule>
+
+
+  <!-- VPC Flow -->
+  <!-- Documentation: https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/flow-logs.html -->
```
Comment (jcruzlp) [inline ruleset/rules/0350-amazon_rules.xml:273]: Not sure if this should be here or in the header.

**Review by jcruzlp — CHANGES_REQUESTED** [review]
I'm not sure about the things I said, they're not bad per se, but is not the same in all the file.

**Review by nmkoremblum — CHANGES_REQUESTED** [review]
Review group orders, please.

**Review by nmkoremblum — CHANGES_REQUESTED** [review]
Please, use only one line to set the groups inside the rule.

**Review by JcabreraC — CHANGES_REQUESTED** [review]
Check line breaks between rules. Several times there is a double line break.

**Comment by 72nomada** [comment]
This PR is closed as changes and improvements requested are managed by #11177

**Pull request closed without merging**